src/content/docs/en/guides/styling.mdx

```diff
@@ -325,10 +327,10 @@ import "./make-it-purple.css"
 </div>
 ```
 
-If you make the imported style _more specific_, it will have higher precedence over the scoped style:
+Scoped styles will be overwritten if the imported style is _more specific_. The style with a higher specificity will take precedence over the scoped style:
```

Comment:
```suggestion
Scoped styles will be overwritten if the imported style is more specific. The style with a higher specificity will take precedence over the scoped style:
```
Just also taking the opportunity to remove italics from older docs when we come across them, as it's no longer our style to use them!